scale-tests/spark-batch-workload.json.template

```diff
@@ -2,12 +2,12 @@
   "id": "/spark-batch-workload",
   "backoffFactor": 1.15,
   "backoffSeconds": 1,
-  "cmd": "cd $MESOS_SANDBOX; git clone https://github.com/mesosphere/spark-build.git; cd spark-build/; pwd; git checkout $SPARK_BUILD_BRANCH; python3 -m venv test-env; pip3 install -r tests/requirements.txt; dcos cluster setup https://master.mesos --username=$DCOS_UID --password=$DCOS_PASSWORD --no-check; dcos package install spark --cli --yes; cd scale-tests/; export PYTHONPATH=../spark-testing:../testing; python3 batch_test.py $MESOS_SANDBOX/$SCRIPT_ARGS",
+  "cmd": "cd $MESOS_SANDBOX; git clone https://github.com/mesosphere/spark-build.git; cd spark-build/; pwd; git checkout $SPARK_BUILD_BRANCH; python3 -m venv test-env; pip3 install -r scale-tests/requirements.txt; dcos cluster setup https://master.mesos --username=$DCOS_UID --password=$DCOS_PASSWORD --no-check; dcos package install spark --cli --yes; cd scale-tests/; export PYTHONPATH=../spark-testing:../testing; python3 batch_test.py $MESOS_SANDBOX/$SCRIPT_ARGS",
   "container": {
     "type": "DOCKER",
     "volumes": [],
     "docker": {
-      "image": "mesosphere/spark-build:latest",
+      "image": "susanxhuynh/dcos-commons:spark",
```

Comment:
Since this application only launches the batch workloads, was this docker image change required?

Reply:
Yes, it has to do with the recent change in "requirements.txt" which got removed from the "tests/" directory, and a new "requirements.txt" compatible with the `dcos-commons` image was added in the "scale-tests" directory. So, I had to move this test to a `dcos-commons`-based image as well (otherwise, some requirements would be missing).